client/index.html

```diff
@@ -3,125 +3,126 @@
 <!--[if IE 7]>         <html class="no-js lt-ie9 lt-ie8"> <![endif]-->
 <!--[if IE 8]>         <html class="no-js lt-ie9"> <![endif]-->
 <!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
-  <head>
```

Comment:
This file I would also consider a grunt - style file. The comments are not for usability, but actually dictate sections of the file.